pilot/pkg/proxy/envoy/testdata/h2-gateway.yaml.golden

```diff
@@ -0,0 +1,15 @@
+servers:
+- port:
+    number: 80
+    protocol: http2
+  hosts:
+  - example.com
+- port:
+    number: 443
+    protocol: http2
```

Comment:
did you mean https here?

Reply:
doesn't http2 do https exclusively anyway?